### ISSUE TITLE
Input Time Picker (Accordion Complex) - Inform assistive technologies of changes in content - (2036570515)

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#5633]
### Module: 
 41 Accordion Complex 

### Violation: 
 Inform assistive technologies of changes in content

![image](https://user-images.githubusercontent.com/5023024/202036054-eff4a911-0638-4ea5-9f89-9873dff3ff83.png) 

### WCAG Reference: 
- [4.1.2 Name, Role, Value](https://www.w3.org/WAI/WCAG21/Understanding/name-role-value.html) 

### Instance ID: 
 2036570515 

### Severity: 
 9 

### Description: 
 [Issue]
When users navigate to the "Time of Day" control, a dialog appears that is not conveyed to the assistive technology users.

[User Impact]
When assistive technology users are not informed about the dynamically added content it can be confusing for them.

[Code Reference] 
 ```
<input aria-label="Time of Day" class="" enterkeyhint="" inputmode="" placeholder="" step="60" type="text">
 ``` 

### Note: 
 [Suggestion]
Developers must ensure that when content is added dynamically to the page it is announced for assistive technology users, this can be done by using off screen text.

[Compliant Code Example] 
 ```
<input aria-describedby="input-desc" aria-label="Time of Day" class="" enterkeyhint="" inputmode="" placeholder="" step="60" type="text">
(...)
<div class="sr-only" id="input-desc"> Please fill the Time of Day field using dialog</div>

[CSS]
<style>
.sr-only {
  position: absolute !important;
  height: 1px; 
  width: 1px; 
  overflow: hidden;
  padding: 0 !important;
  border: 0! important;
  white-space: nowrap !important;
  clip: rect(1px 1px 1px 1px) !important; /* IE6, IE7 */
  clip: rect(1px, 1px, 1px, 1px) !important;
  clip-path: inset(50%) !important;
}
</style>
 ```
 

### Media Type: 
 ARIA and Dynamic Content 

### Additional Resources: 
- Report Source: Product accessibility evaluation conducted on the Calcite Design System in August, 2022.

**Comment by geospatialem** [comment]
Pertains to the `input-time-picker` component, which should provide AT users context when the component is `open`.

![image](https://user-images.githubusercontent.com/5023024/202036054-eff4a911-0638-4ea5-9f89-9873dff3ff83.png)

**Comment by geospatialem** [comment]
Duplicate of #5611
